pages/dashboard/index.tsx

```diff
@@ -9,4 +9,16 @@ const AggregateDashboard = () => {
 };
 
 export default AggregateDashboard;
-export const getServerSideProps = withSites();
+export const getServerSideProps = withSites({
+  async getServerSideProps({ siteList }) {
+    if (siteList.site_list.length === 1) {
+      return {
+        redirect: {
+          permanent: false,
+          destination: `/dashboard/${siteList.site_list[0].site_uuid}`,
+        },
+      };
+    }
+    return { props: { siteList } };
+  },
+});
```

Comment:
Think a better solution might be to display the `Dashboard` component with the `site_uuid` set to the ID of the only site—wouldn't need to redirect and would keep the routes simple for just one site.

Reply:
The reason I did this was because the view we'll have for a site dashboard vs. aggregate dashboard might very different. We could either load a different component, or more likely just pass a prop to the component. In that case, we really want to just treat someone who has one site's "aggregate dashboard" as their normal dashboard, and we can get that by just showing them the single site dashboard page. Then, linking to `/dashboard` in the sidebar works if we redirect them. Just some abstractions that I think make sense, lmk your thoughts.